core/src/main/scala/org/apache/spark/rpc/netty/NettyStreamManager.scala

```diff
@@ -19,6 +19,7 @@ package org.apache.spark.rpc.netty
 import java.io.File
 import java.util.concurrent.ConcurrentHashMap
 
+import org.apache.spark.SparkContext
```

Comment:
Was this intentional? I would have guessed this wouldn't compile without this import before if it were needed.